apps/change-log.md

```diff
@@ -14,6 +14,20 @@ Of course, we will continue to inform you if there are any breaking changes to t
 
 <hr>
 
+## 2017-12-08
+
+### Changes with software release 7.17.0
+
+#### <i class="fa fa-plus"></i> New
+
+* [`GET`orders/{orderId}/documents](https://developer.epages.com/apps/api-reference/get-shopid-orders-orderid-documents.html)
+
+#### <i class="fa fa-pencil"></i> Update
+
+* Added attributes `documentType` and `documentNumber` to data type [`order`](https://developer.epages.com/apps/data-types#order).
```

Comment:
Even though a document structure is similar to an order structure because documents are copies (at a given time) of the order they belong to, I'd distinguish document data type from order data type because orders contain documents:
an order can hold different documents, each one of them of a specific "documentType" (invoice / credit note) and with a different "documentNumber".